bin/main

```diff
@@ -1,2 +1,78 @@
 #!/usr/bin/env ruby
-puts 'Hello, World!'
+hash = { 1 => 1, 2 => 2, 3 => 3, 4 => 4, 5 => 5, 6 => 6, 7 => 7, 8 => 8, 9 => 9 }
+SEPARATOR = '-----------'.freeze
+
+def display_board(hash)
+  puts [" #{hash[1]} | #{hash[2]} | #{hash[3]} "]
+  puts SEPARATOR
+  puts [" #{hash[4]} | #{hash[5]} | #{hash[6]} "]
+  puts SEPARATOR
+  puts [" #{hash[7]} | #{hash[8]} | #{hash[9]} "]
+end
+
+def place_move(move, player_two, hash)
+  move = move
+  until ((1..9).include? move) && (hash.value? move)
+    puts 'Invalid move'
+    puts 'Write another number between 1 and 9'
+    move = gets.chomp.to_i
+  end
+
+  hash[move] = 'X'
+  display_board(hash)
+  puts "#{player_two} it's your turn. Choose one number between 1 and 9."
+end
+
+# Loop
+def loop_game(name_one, name_two, hash)
+  count = 0
+
+  until count == 4
```

Comment:
Your game loop should be based on a condition that person won or lose, while you are technically using a variable to terminate the loop. This is an out of nowhere terminator.

Please remember that this main file does not have to run (Can even be done with comments), it's simply a way ensuring that you thought about everything listed in the previous review.